### PR TITLE
`seriesRLC` init

### PR DESCRIPTION
**Pull request by Lakshmikanth2001**
resolving #104

### CODE REVIEW

**Comment by Lakshmikanth2001** [comment]
white space is present that's the reason `pydocstyle` is failing if you are ok with the function let me know

**Comment by Lakshmikanth2001** [comment]
frequencyResponse

File: electricpy/visu.py [hunk 1]
```diff
@@ -12,16 +12,17 @@
 import numpy as _np
 import matplotlib as _matplotlib
 import matplotlib.pyplot as _plt
+import matplotlib.colors as mcolors
```
Comment (engineerjoe440) [inline electricpy/visu.py]: Please prefix with an underscore (_mcolors)

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Just the one comment, then let's slam this in!

File: electricpy/visu.py (hunk 1 above)
Comment (Lakshmikanth2001) [reply electricpy/visu.py]: ok

File: test/test_opamp.py
Comment (engineerjoe440) [inline test/test_opamp.py:1]: I don't think this should be added as part of this pull-request. Was probably part of some of the other work you were doing for #70 ?

Can we remove this from this PR, or expand its scope of work to include what you're working on for #70 ?

File: test/RLC_frequency_response.png
Comment (engineerjoe440) [inline test/RLC_frequency_response.png:1]: I don't think this image is really intended to be here. I love the figure, but I don't think it's actually part of the material that should be added, here. Right?

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]

File: electricpy/visu.py (hunk 1 above)
Comment (engineerjoe440) [reply electricpy/visu.py]: did you just end up removing this line?

Comment (Lakshmikanth2001) [reply electricpy/visu.py]: that was an unused import

File: test/test_opamp.py
Comment (Lakshmikanth2001) [reply test/test_opamp.py:1]: yes you are right let me uncheck those files

File: test/RLC_frequency_response.png
Comment (Lakshmikanth2001) [reply test/RLC_frequency_response.png:1]: yes let me fix that

**Review by engineerjoe440 — APPROVED** [review]

**Pull request merged**